mmv1/products/parallelstore/Instance.yaml

```diff
@@ -36,26 +35,28 @@ examples:
 properties:
   - !ruby/object:Api::Type::String
     name: name
-    description: "The resource name of the instance, in the format\n`projects/{project}/locations/{location}/instances/{instance_id}` "
+    description: "Identifier. The resource name of the instance, in the format\n`projects/{project}/locations/{location}/instances/{instance_id}` "
     output: true
   - !ruby/object:Api::Type::String
     name: description
-    description: 'The description of the instance. 2048 characters or less. '
+    description: 'Optional. The description of the instance. 2048 characters or less. '
   - !ruby/object:Api::Type::String
     name: state
-    description: "The instance state. \n Possible values:\n STATE_UNSPECIFIED\nCREATING\nACTIVE\nDELETING\nFAILED"
+    description: "Output only. The instance state. \n Possible values:\n STATE_UNSPECIFIED\nCREATING\nACTIVE\nDELETING\nFAILED\nUPGRADING"
```

Comment:
These descriptions are not necessary. We already account for Optional and Output only labeling in our documentation. Can we remove these additions?

See the [generated markdown](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-11382-old..auto-pr-11382#diff-2cdf9cc03326323f4d012ad019fdae0dd18817a8b960e56925ea9014585dee33) - can see the duplication of this information there.

Reply:
Removed. Requesting someone else for a quick review as we need this cl in before next cut